pcdsdevices/epics/mps.py

```diff
@@ -134,3 +134,52 @@ def mps_factory(clsname, cls,  *args, mps_prefix, veto=False,  **kwargs):
     comp = FC(MPS, mps_prefix, veto=veto)
     cls  = type(clsname, (cls,), {'mps' : comp})
     return cls(*args, **kwargs)
+
+
+def mustBeOpenLogic(mps_A, mps_B):
```

Comment:
Can you make these `must_be_open_logic`? We typically don't use camel case

Reply:
Chiming in to say that this is part of the pep8 specification for python style: https://www.python.org/dev/peps/pep-0008/#function-names